apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1538,9 +1537,7 @@
   };
 
   MockNavigatormozMobileMessage.retrieveMMS = function(id) {
-    var request = {
-      error: null
-    };
+    var deferred = Utils.Promise.defer();
```

Comment:
Mmmm, what is ```navigator_moz_mobilemessage_shim.js```? I don't see this file anywhere? I believe we'll have 404 in browser....